pkg/kafka/policy.go

```diff
@@ -281,9 +222,42 @@ func (req *RequestMessage) ruleMatches(rule api.PortRuleKafka) bool {
 // rules. The function will return true if the policy allows the message,
 // otherwise false is returned.
 func (req *RequestMessage) MatchesRule(rules []api.PortRuleKafka) bool {
+	reqTopicsMap := map[string]bool{}
```

Comment:
```
topics := req.GetTopics()
reqTopicsMap := make(map[string]bool, len(topics))
```

Reply:
https://blog.golang.org/go-maps-in-action
```
The same syntax may be used to initialize an empty map, which is functionally identical to using the make function:

m = map[string]int{}
```
So declaring and empty map and then adding to it 
```
reqTopicsMap := map[string]bool{} 
```
is functionally similar to doing a `make`. I can add it if you like it this way. But semantically both are equivalent.

Reply:
No it isn't the same. `map[string]bool{}` creates a map of capacity zero. `make(map[string]bool, len(topics))` creates a map of capacity `len(topics)`, so this method won't need to keep re-allocating memory for this map.

Reply:
ah.. ok thanks for clarifying